packages/wonder-blocks-tooltip/src/components/__docs__/tooltip.stories.js

```diff
@@ -102,6 +141,10 @@ export const AnchorInScrollableParent: StoryComponentType = () => (
 );
 
 AnchorInScrollableParent.parameters = {
+    // Disable Chromatic because it only shows the trigger element.
+    chromatic: {
+        disableSnapshot: true,
+    },
```

Comment:
**question:** How does Chromatic know when to take the snapshot of the stories using `.play()`?  Does it wait until `.play()` finishes before snapshotting?

Reply:
That's correct! https://www.chromatic.com/docs/hoverfocus#make-your-stories-interactive

> Chromatic awaits play function execution before taking a snapshot.